torch/fx/graph.py

```diff
@@ -291,16 +291,15 @@ def __exit__(self, type, value, tb):
 
 class _node_list:
     def __init__(self, graph: "Graph", direction: str = "_next"):
-        assert direction in ["_next", "_prev"]
+        assert direction in ("_next", "_prev")
         self.graph = graph
         self.direction = direction
```

Comment:
This is pad code smell, the self.direction should really be a private StrEnum

Reply:
I worry this is a BC surface and don't want to break user code.

Reply:
Agreed, @jansel that's why I would recommend the literal approach below that would at least use static type checking